jbpm-installer/src/main/resources/db/upgrade-scripts/sybase/jbpm-6.5-to-7.0.sql

```diff
@@ -19,20 +24,27 @@ create table CaseRoleAssignmentLog (
     roleName varchar(255) null,
     type int not null,
     primary key (id)
-) lock datarows;
+) lock datarows
+go
 
 alter table CaseIdInfo 
-    add constraint UK_CaseIdInfo_1 unique (caseIdPrefix);
+    add constraint UK_CaseIdInfo_1 unique (caseIdPrefix)
+go
     
-ALTER TABLE NodeInstanceLog ADD COLUMN referenceId bigint null;
-ALTER TABLE NodeInstanceLog ADD COLUMN nodeContainerId varchar(255) null;      
+ALTER TABLE NodeInstanceLog ADD COLUMN referenceId bigint null
```

Comment:
One more thing, please delete COLUMN from here.
With this script I am getting
```
Query execution failed

Reason:
SQL Error [102] [37000]: Incorrect syntax near 'bigint'.
```

If I remove the COLUMN the command gets processed correctly. Also on other places the column addition is done without COLUMN, like https://github.com/kiegroup/jbpm/pull/1430/files#diff-8b4283656397076d77968b57e8a750d3R3

Reply:
updated